docs/index.md

```diff
@@ -52,3 +52,6 @@ Building our cloud future has never been this simple.
   * [MySQL Exporter](prometheus-mysql-exporter.md)
   * [RabbitMQ Exporter](prometheus-rabbitmq-exporter.md)
   * [Openstack Exporter](prometheus-openstack-metrics-exporter.md)
+
+## Exposing Flex Services
```

Comment:
the docs have been significantly reworked, can we put this into our nav flow? Maybe somewhere here https://github.com/rackerlabs/genestack/blob/main/mkdocs.yml#L115 showing the two options we have, until we can eliminate the legacy ingress controller.